source/draw/gpu/opengl/backend/gles3/Gles3ShaderProgram.ooc

```diff
@@ -86,6 +87,31 @@ Gles3ShaderProgram: class extends GLShaderProgram {
 		glUniform4f(glGetUniformLocation(this _backend, name), value x, value y, value z, value w)
 		version(debugGL) { validateEnd("ShaderProgram setUniform~FloatPoint4D") }
 	}
+	setUniform: override func ~ColorBgr (name: String, value: ColorBgr) {
+		version(debugGL) { validateStart("ShaderProgram setUniform~ColorBgr") }
+		glUniform3f(glGetUniformLocation(this _backend, name), (value blue as Float) / 255.0f, (value green as Float) / 255.0f, (value red as Float) / 255.0f)
```

Comment:
So how about this, `ShaderProgram setUniform` overloads (and same for integers):
```ooc
setUniform: func (value: Float)
setUniform: func (value1, value2: Float)
setUniform: func (value1, value2, value3: Float)
setUniform: func (value1, value2, value3, value4: Float)
setUniform: func (values: Float*, count: Int)
```

In OpenGLMap use() conversions from Imint standard to OpenGL standard can be added:
```ooc
			program := this program
			if (value instanceOf?(Cell)) {
				cell := value as Cell
				match (cell T) {
					case Int => program setUniform(key, cell as Cell<Int> get())
					case IntPoint2D => program setUniform(key, cell as Cell<IntPoint2D> get())
					case IntPoint3D => program setUniform(key, cell as Cell<IntPoint3D> get())
					case IntSize2D => program setUniform(key, cell as Cell<IntSize2D> get())
					case IntSize3D => program setUniform(key, cell as Cell<IntSize3D> get())
					case Float => program setUniform(key, cell as Cell<Float> get())
					case FloatPoint2D => program setUniform(key, cell as Cell<FloatPoint2D> get())
					case FloatPoint3D => program setUniform(key, cell as Cell<FloatPoint3D> get())
					case FloatPoint4D => program setUniform(key, cell as Cell<FloatPoint4D> get())
					case FloatSize2D => program setUniform(key, cell as Cell<FloatSize2D> get())
					case FloatSize3D => program setUniform(key, cell as Cell<FloatSize3D> get())
					case FloatTransform2D => program setUniform(key, cell as Cell<FloatTransform2D> get())
					case FloatTransform3D => program setUniform(key, cell as Cell<FloatTransform3D> get())
					case => Debug raise("Invalid cover type in OpenGLMap use!")
				}
			}
			else
				match (value) {
					case image: OpenGLPacked => {
						image backend bind(textureCount)
						program setUniform(key, textureCount)
						textureCount += 1
					}
					case image: OpenGLVolumeMonochrome => {
						image _backend bind(textureCount)
						program setUniform(key, textureCount)
						textureCount += 1
					}
					case => Debug raise("Invalid object type in OpenGLMap use: %s" format(value class name))
				}
```